### ISSUE TITLE
FontConvertible 'Expression implicitly coerced from 'UIFont?' to Any

### ISSUE THREAD

**Issue by nickprkins** [SwiftGen/SwiftGen#631]
Since updating my project to Swift 5 and Xcode 10.2 I am finding these warnings everywhere.

> Expression implicitly coerced from `UIFont?` to `Any`
> Provide a default value to avoid this warning
> Force-unwrap the value to avoid this warning

Seems excessive for using a font in my assets.  
Here is one piece of code showing the problem.

let attributedString = NSMutableAttributedString(string: "Login", attributes: [
            .font: UIFont(font: FontFamily.OurFont.regular, size: 28), <- warning here
            .paragraphStyle: paragraphStyle
            ])

Using SwiftGen 6.1.0

**Comment by djbe** [comment]
This is a known issue, and will be dealt with in #600.

The fix is discussed there, and you can temporarily use it in a custom template until we release the next SwiftGen version.

**Comment by djbe** [comment]
Do note that, when using the Swift 5 template, you'll need to call the `font(size:)` function on a SwiftGen font object (`FontConvertible`). The `UIFont` initialiser is failable, and'll return an optional.

So your code would become:
```swift
let attributedString = NSMutableAttributedString(string: "Login", attributes: [
  .font: FontFamily.OurFont.regular.font(size: 28),
  .paragraphStyle: paragraphStyle
])
```

**Comment by djbe** [comment]
Fixed in SwiftGen 6.2.0, should be available on CocoaPods (and other package managers) very soon!
